examples/resources/fabric_precision_time/ntp_time_service.tf

```diff
@@ -0,0 +1,17 @@
+resource "equinix_fabric_precision_time" "ntp" {
```

Comment:
There's more of a tradeoff for these examples than the data source example, but you could merge these resource examples into a single file, with `# comments` describing each block if necessary, put the merged file at `examples/resources/equinix_fabric_precision_time/resource.tf` and remove the custom doc template for the resource as well.

Reply:
I tried out this documentation adjustment you suggested for Data Source and Resource and didn't get the results that were expected. I still feel the templates are necessary based on what Fabric team expects before the schema information, and the examples weren't included following that pattern.